managedplugin/plugin.go

```diff
@@ -166,6 +168,19 @@ func NewClient(ctx context.Context, typ PluginType, config Config, opts ...Optio
 	return &c, nil
 }
 
+func (c *Client) ConnectionString() string {
+	tgt := c.Conn.Target()
+	switch c.registry {
+	case RegistryGrpc:
+		return "grpc://" + tgt
+	case RegistryLocal:
+		return "unix://" + tgt
+	case RegistryGithub:
+		return "unix://" + tgt
```

Comment:
where you able to test it? that it works on connection creation? I don't remember if it's `unix://` or `unix:///`

Reply:
I tested the `unix` one yeah, haven't tested `grpc://` yet, let me do that now

Reply:
yeah actually `grpc://` doesn't work, it says `too many colons in address`...

Reply:
Ok fixed. In the `grpc://` case it should have no prefix, `unix://` is treated as a special value by the Go GRPC client.